pkg/operator/ceph/object/objectstore.go

```diff
@@ -736,7 +737,13 @@ func CreatePools(context *Context, clusterSpec *cephv1.ClusterSpec, metadataPool
 	}
 
 	// get the default PG count for rgw metadata pools
-	metadataPoolPGs, err := config.GetMonStore(context.Context, context.clusterInfo).Get("mon.", "rgw_rados_pool_pg_num_min")
+	var err error
+	var metadataPoolPGs string
+	if context.clusterInfo.CephVersion.IsAtLeast(cephvers.Quincy) {
```

Comment:
In what version exactly was that merged? Based on the timeline of that ceph merge, I'm guess 17.2.2?
```suggestion
	if context.clusterInfo.CephVersion.IsAtLeast(cephver.CephVersion{Major: 17, Minor: 2, Extra: 2}) {
```